packages/image_picker/lib/image_picker.dart

```diff
@@ -50,12 +53,22 @@ class ImagePicker {
       throw ArgumentError.value(maxHeight, 'maxHeight cannot be negative');
     }
 
+    if (null == imageQuality) {
```

Comment:
let's make imageQuality optional to reduce unwanted computations. 
```dart
assert(imageQuality == null || (imageQuality >= 0 && imageQuality <= 100));
```

Reply:
do we need to make imageQuality mandatory? now if imageQuality gets null from a user, replacing it with 100 does not have any impact on image quality, it will return the image as it is.

Reply:
No, we don't need to make the imageQuality mandatory. However, I'd like to keep the parameters sent to the platform consistent with what user has passed in the method call, and let the underlying platform handle the case when imageQuality is null.
So the method channel params will look like 
{
   source:"camera",
   imageQuality:null,
}

And from the platform (iOS for example) . 
```objc
NSNumber *quality = @(1);
if ([argument[@"imageQuality"] isKindOfClass[NSNumber class]]) {
   quality = [argument[@"imageQuality] floatValue]/100;
}
```

It is just my personal opinion. Your code works perfectly fine as it is now!

Reply:
done, added checks in Android and iOS to check for null